pkg/windows/windows.go

```diff
@@ -456,7 +464,7 @@ func (vm *windows) Deconfigure() error {
 	return nil
 }
 
-func (vm *windows) Configure() error {
+func (vm *windows) Prepare() error {
```

Comment:
I don't see the reason to separate this into `Prepare` and `Bootstrap`. Is there ever a time we'd need to prepare a node and not bootstrap it?

This change makes the Windows interface more clunky if that is not the case

Reply:
> Is there ever a time we'd need to prepare a node and not bootstrap it?

Not that I can think of. You're right, I consolidated them